README.md

```diff
@@ -2,7 +2,7 @@
 
 
 # OWASP Mobile Security Testing Guide [![Twitter Follow](https://img.shields.io/twitter/follow/OWASP_MSTG.svg?style=social&label=Follow)](https://twitter.com/OWASP_MSTG)
-[![Creative Commons License](https://i.creativecommons.org/l/by-sa/3.0/88x31.png)](http://creativecommons.org/licenses/by-sa/3.0/ "CC-by-SA 3")
+[![Creative Commons License](https://licensebuttons.net/l/by-sa/4.0/88x31.png)](https://creativecommons.org/licenses/by-sa/4.0/ "CC BY-SA 4.0")
```

Comment:
Is this the same in the masvs, the readme and the OWASP wiki?

Reply:
let me check on that :-)

Reply:
Thanks for merging! Now MASVS, MSTG and Wiki is all CC 4.0

Reply:
Can you update the frontispiece in masvs and mstg as well then?